### ISSUE TITLE
Style/SymbolArray breaks indentation for multiline arrays

### ISSUE THREAD

**Issue by aried3r** [bbatsov/rubocop#5225]
## Expected behavior

Given this code:

```Ruby
class Example < ApplicationRecord
  has_paper_trail on: [:create, :update], only: [
    :foo,
    :bar,
    :baz
  ]
end
```

I'd expect rubocop to produce the following when running `--auto-correct`:

```ruby
class Example < ApplicationRecord
  has_paper_trail on: %i[create update], only: %i[
    foo
    bar
    baz
  ]
end
```

## Actual behavior

Running auto correct produces the following code, ignoring any kind of indentation:

```ruby
class Example < ApplicationRecord
  has_paper_trail on: %i[create update], only: %i[
foo 
bar 
baz]
end
```

What isn't easily visible here is that the cop also created trailing whitespace where the commas were before after `foo` and `bar`.

## Steps to reproduce the problem

Run `rubocop -a --only Style/SymbolArray example.rb`

```ruby
# example.rb

class Example < ApplicationRecord
  has_paper_trail on: %i[create update], only: %i[
    foo
    bar
    baz
  ]
end
```

## RuboCop version

```
$ rubocop -V
0.52.0 (using Parser 2.4.0.2, running on ruby 2.4.2 x86_64-darwin16)
```

**Comment by koic** [comment]
This issue is duplicated with #5409, and fixed by #5410. So this issue can be closed.
